dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -839,6 +839,7 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 														CAPIArticle.filterKeyEvents
 													}
 													id="filter-toggle-desktop"
+													tooltip={true}
```

Comment:
This means you can remove each of these too

Reply:
Done, thanks!